server/api/src/rsDataObjRename.cpp

```diff
@@ -47,8 +47,6 @@
 #include <cstring>
 #include <chrono>
 
-using logger = irods::experimental::log;
```

Comment:
Shouldn't this be `using log_api = irods::experimental::log::api;` instead of being removed?

Reply:
There was a name clash that developed for some reason. I'll change the style over :)